js/ui/components/clubhouse.js

```diff
@@ -114,12 +115,33 @@ var ClubhouseNotificationSource = new Lang.Class({
     },
 });
 
+var NotifiableButton = new Lang.Class({
+    Name: 'NotifiableButton',
+    Extends: St.Button,
+
+    _init: function(params) {
+        this.parent(params);
+
+        this.connect('clicked', () => {
+            this._stopPulsing();
+        })
```

Comment:
Since this is the button implementation, we shouldn't connect to this button. Instead, we should stop the highlighting in a `clicked` implementation. However, I think that for now we should simply stop pulsating when the Clubhouse is to be opened (already handled in the `ClubhouseComponent`).